src/mame/audio/vicdual.cpp

```diff
@@ -558,14 +558,34 @@ void borderline_audio_device::device_add_mconfig(machine_config &config)
 		.set_source(NETLIST_NAME(brdrline))
 		.add_route(ALL_OUTPUTS, *this, 1.0);
 
-	NETLIST_LOGIC_INPUT(config, m_input_line[7], "GUN_TRG.IN", 0);
+#if 0
+2020-10-05 by 'beta-tester'
+  |                         |                  ||assignment   |assignment   |
+  |brdrline                 |starrkr           ||NL SOUND     |plausibility |note
+--+-------------------------+------------------++-------------+-------------+----
+0 |                         |fire, jeep_field  ||POINT_TRG.IN |             |
+1 |next_sector, hit_rocket2 |hit_animal        ||HIT_TRG.IN   |+            |
+2 |                         |fire, next_sector ||WALK_TRG.IN  |             |
+3 |                         |fire              ||CRY_TRG.IN   |             |
+4 |jeep_field               |                  ||ANIMAL_TRG.IN|+++          |see note 1
+5 |fire                     |                  ||GUN_TRG.IN   |++           |
+6 |jeep_path                |jeep_path?        ||JEEP_ON.IN   |++           |
+7 |hit_animal               |                  ||EMAR_TRG.IN  |+            |
+
+note 1: as far as i remember it was triggered more often while crawling through the field at sector2 & 3,
+ issue in schematic/netlist?
+       or trigger?
+       or were the acrade what i played in the past a bootleg
+       or is it only in my head?
+#endif
+	NETLIST_LOGIC_INPUT(config, m_input_line[5], "GUN_TRG.IN", 0);
 	NETLIST_LOGIC_INPUT(config, m_input_line[6], "JEEP_ON.IN", 0);
```

Comment:
Can you please create aliases in the netlist called I_SOUND_0, I_SOUND_1, etc, which will directly map to the m_input_line[0], m_input_line[1], etc? This way, we can leave the C code alone and just change the mapping in the netlist itself (and I would move the above comment to the netlist). When we add netlists for more vicdual games, we can use a generic sound device that doesn't understand the mapping and just connects the input bits to generic-named inputs.

Reply:
done in commit 566ff67b4665382aeab4a75134dc86a7ff684b7c below